src/accordion/tp-accordions-collapse-all.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * TPAccordionsCollapseAllElement.
+ */
+import { TPAccordionsElement } from './tp-accordions';
+
+export class TPAccordionsCollapseAllElement extends HTMLElement {
+	connectedCallback() {
+		this.addEventListener( 'click', () => this.collapseAllAccordions() );
+	}
+
+	collapseAllAccordions() {
+		const accordions: TPAccordionsElement | null = this.closest( 'tp-accordions' );
```

Comment:
@sagarjadhav From here, we should only be setting an attribute on the parent

Reply:
@junaidbhura I believe we are setting to 'tp-accordions' here. So the parent controls the children.